### ISSUE TITLE
K-CAP red links in locations

### ISSUE THREAD

**Issue by WolfgangFahl** [tholzheim/orapi#26]
https://confident.dbis.rwth-aachen.de/or/index.php?title=K-CAP has red links - will republishing fix this?
Why do the red links not show in the https://confident.dbis.rwth-aachen.de/or/index.php?title=List_of_Locations?

**Comment by WolfgangFahl** [comment]
same with https://confident.dbis.rwth-aachen.de/or/index.php?title=ICML

**Comment by tholzheim** [comment]
Yes it does. I republished the series and now the red links are gone.

The [List of Locations](https://confident.dbis.rwth-aachen.de/or/index.php?title=List_of_Locations) queries for the location entities since locations with a red link are not instantiated they can not be queried propertly.
We can either query for the value that assigns the location e.g. **Has location country** but then we can not aggregate the values or filter for the red links properly.

the Redlinks are listed here but can not be properly queried:

- https://confident.dbis.rwth-aachen.de/or/index.php?title=Property:Has_location_country
- https://confident.dbis.rwth-aachen.de/or/index.php?title=Property:Has_location_city
- https://confident.dbis.rwth-aachen.de/or/index.php?title=Property:Has_location_region
- https://confident.dbis.rwth-aachen.de/or/index.php?title=Special:WantedPages&limit=500&offset=0

**Comment by WolfgangFahl** [comment]
https://confident.dbis.rwth-aachen.de/or/index.php?title=US/CA/Marina has still a read link to US/CA ... the check IMHO needs to go to the country level. The levels above that do not need to be automated but it should be warned if the region entry or  page is missing so that the curation is eased by clicking link to these page (please display missing links  in red and may a link to a a page with missing information in yellow.)

**Comment by tholzheim** [comment]
Yeah, forgot that  the region template param name is **State** and not **Region** and that in the publishing phase we do not work with the normalized properties (see #27)
